lib/logic/models/pull-request.js

```diff
@@ -137,4 +149,24 @@ PullRequest.prototype.getActionsToDo = function() {
     });
 };
 
+PullRequest.prototype.isLGTM = function() {
+    if (!this.getComments().length) {
+        return false;
+    }
+    var lastComment = this.getComments()[this.getComments().length - 1];
+    return lastComment.body.indexOf("LGTM") >= 0;
```

Comment:
I would prefer it if you used `getBody()` rather than via the property. Technically `this.body` *may* be `null` as I don't trust the GH APIs to not be silly, whereas `getBody()` would ideally (it doesn't currently) default to an empty string to make the API nicer so you can actually call things like `indexOf()` without erroring.